src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -31,6 +32,7 @@ See the LICENSE file in the project root for more information.
   </ItemGroup>
 
   <ItemGroup>
+    <NativeLibrary Condition="'$(IlcMultiModule)' == 'true' and $(NativeCodeGen) == ''" Include="$(FrameworkLibPath)\libframework.a" />
```

Comment:
Do we need to duplicate line per each platform? Could we just include `$(SharedLibrary)` from the `.targets` file?